doc/examples/ex51/ex51.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+# GMT EXAMPLE 51
+#
+# Purpose:     Illustrate usage of OpenStreetMap coastlines in GMT
+# GMT modules: convert, select, coast, plot
+# GDAL progs:  ogr2ogr
+
+
+# First we convert the shapefile to something more GMT friendly. Here we make it
+# into a large ASCII file. Human-readable but not very efficient. GDALs ogr2ogr
```

Comment:
```suggestion
# into a large ASCII file. Human-readable but not very efficient. GDAL's ogr2ogr
```

Reply:
German Grammar in English text … thanks for catching it.